pasta/augment/import_utils_test.py

```diff
@@ -116,6 +116,41 @@ def test_split_nested_imports(self):
         self.fail('Failed while executing case:\n%s\nCaused by:\n%s' % 
                   (src, traceback.format_exc()))
 
+class GetUnusedImportsTest(test_utils.TestCase):
+
+  def test_normal_imports(self):
+    src = """\
```

Comment:
tbh I would just put this on a single line with `\n`'s, there's not much gained here by spreading it out.

Reply:
As discussed removed the line breaks.